main.go

```diff
@@ -153,6 +153,14 @@ func main() {
 		setupLog.Error(err, "unable to create controller", "controller", "Component")
 		os.Exit(1)
 	}
+	if err = (&controllers.PortalReconciler{
+		Client:   mgr.GetClient(),
+		Scheme:   mgr.GetScheme(),
+		Recorder: mgr.GetEventRecorderFor("repository-reconcile"),
```

Comment:
Should change this to `portal-reconcile`

Reply:
Thank you! The name is now changed to "portal-reconcile".